app/views/logistics_view.py

```diff
@@ -33,8 +29,9 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
     emp_job_role = {'Job Role ID': current_job_role['uniqueRoleId'],
                     'Job Role': current_job_role['jobRole'],
-                    'Badge Number': get_emp_info('idBadgeNo'),
+                    'Job Role Type': current_job_role['jobRoleType'],
```

Comment:
I'm not sure this has anything to do with the 'job title short' mentioned in FWMT-2685

Reply:
Have removed this line